src/components/Drawer/styles.js

```diff
@@ -1,50 +1,99 @@
 import styled from '@emotion/styled'
 import { createMuiTheme } from '@material-ui/core/styles'
 import { theme as localTheme } from '~/utils/styles'
-import Text from '~/components/Utilities/Text'
 
-export const DrawerItem = styled(Text)`
-  /* keeping it for later when I will add the text for the drawer menu as an example */
+export const IconContainer = styled.div`
+  /* used for the onClickFunction */
 `
+export const DrawerSocialMedia = styled.div`
+  @media (min-width: ${localTheme.breakpoints.md}) {
+    max-width: 400px;
+  }
+
+  .Social-Media-Container {
+    margin-bottom: 0;
+
+    .SocialMedia__Logo {
+      width: 147px;
+
+      @media (min-width: ${localTheme.breakpoints.s}) {
+        width: 127px;
+      }
+
+      @media (min-width: ${localTheme.breakpoints.md}) {
+        width: 219px;
+      }
+    }
+
+    .Social__Icon-Container {
+      justify-content: flex-start;
+    }
+  }
+
+  .Social__Logo-Container {
+    margin-left: 0;
+    margin-right: 16px;
+    padding-right: 10px;
+
+    @media (min-width: ${localTheme.breakpoints.md}) {
+      margin-right: 27px;
+    }
+  }
+`
+
+export const DrawerHeaderContent = styled.div`
+  padding-left: 34px;
+  margin-top: -13px;
 
-export const DrawerButtons = styled.div`
+  .Drawer__Menu-Title {
+    margin-bottom: 14px;
+  }
+`
+export const DrawerContainer = styled.div`
   display: flex;
-  justify-content: flex-end;
   align-items: center;
-  height: 24px;
-  position: absolute;
-  left: 56%;
-  top: 5%;
+  justify-content: flex-end;
+  padding-right: 31px;
+  padding-top: 28px;
 
   @media (min-width: ${localTheme.breakpoints.md}) {
-    left: 0;
-    top: 5%;
-    display: flex;
-    justify-content: inherit;
-    width: 100%;
-    justify-content: flex-end;
-    padding-right: 103px;
+    padding-right: 45;
+    padding-top: 47;
   }
 `
 
-export const IconContainer = styled.div`
-  cursor: pointer;
-  position: absolute;
-  left: 84.06%;
-  right: 9.69%;
-  top: 5.28%;
-  bottom: 91.2%;
+export const DrawerMenuTextContainer = styled.div`
+  margin-bottom: 116px;
+`
 
+export const DrawerMenuText = styled.div`
+  margin-bottom: 5px;
+  span {
+    display: none;
+  }
   @media (min-width: ${localTheme.breakpoints.md}) {
-    left: 90%;
+    margin-bottom: 0;
+
+    span {
```

Comment:
I see that you use span for styling here.

I'd suggest you create another emotion container like `IconContainer` and style that instead. Will be cleaner and will avoid potential conflicts